demo 2/src/main/java/com/example/demo/repository/table/ItemRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demo.repository.table;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ItemRepository extends JpaRepository<ItemTable, Long> {
+    @Query(value = "select * from item_table where url = :url and phone = :phone ", nativeQuery = true)
+    public List<ItemTable> findByUrl(String url, Long phone);
```

Comment:
search by keyword missing.
Ex:
Searching `face` should return `www.facebook.com`